frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelineRunTable.ts

```diff
@@ -46,9 +46,9 @@ class PipelineRunTable {
   mockGetRuns(runs: PipelineRunKFv2[], times?: number) {
     return cy.intercept(
       {
-        times,
         method: 'POST',
         pathname: '/api/proxy/apis/v2beta1/runs',
+        ...(times && { times }),
```

Comment:
Interesting, is this just because `times` can be 0 and be weird for the intercept?

Reply:
cy.intercept requires a number value for `times`, so passing undefined as I was doing without this spread led to cypress throwing an error stating as much.